lib/livebook/k8s/kubeconfig.ex

```diff
@@ -0,0 +1,7 @@
+defmodule Livebook.K8s.Kubeconfig do
```

Comment:
I think we should roll with the default kubeconfig. I don't see a reason for supporting LIVEBOOK_KUBECONFIG because:

1. setting KUBECONFIG or LIVEBOOK_KUBECONFIG for the desktop or the CLI is the same amount of work
2. setting KUBECONFIG has the benefit that, if you want to use Kubereq inside the runtime, it will just work too

Plus, we probably want to support reading from `~/.kube/config` anyway,  no? This means we can also remove `:pluggable` from the .formatter.exs file :)

Reply:
> This means we can also remove :pluggable from the .formatter.exs file 

Because we can use `https://github.com/mruoss/kubereq/blob/main/lib/kubereq/kubeconfig/default.ex` you mean?

Reply:
Correct!